internal/cmd/kafka/command_link_create.go

```diff
@@ -80,10 +95,16 @@ func (c *linkCommand) newCreateCommand() *cobra.Command {
 	cmd.Flags().String(sourceBootstrapServerFlagName, "", `Bootstrap server address of the source cluster. Can alternatively be set in the configuration file using key "bootstrap.servers".`)
 	cmd.Flags().String(destinationClusterIdFlagName, "", "Destination cluster ID for source initiated cluster links.")
 	cmd.Flags().String(destinationBootstrapServerFlagName, "", `Bootstrap server address of the destination cluster for source initiated cluster links. Can alternatively be set in the configuration file using key "bootstrap.servers".`)
-	cmd.Flags().String(sourceApiKeyFlagName, "", "An API key for the source cluster. For links at destination cluster, this is used for remote cluster authentication. For links at source cluster, this is used for local cluster authentication. "+authHelperMsg)
-	cmd.Flags().String(sourceApiSecretFlagName, "", "An API secret for the source cluster. For links at destination cluster, this is used for remote cluster authentication. For links at source cluster, this is used for local cluster authentication. "+authHelperMsg)
+	cmd.Flags().String(remoteClusterIdFlagName, "", "Remote cluster ID for bidirectional cluster links.")
```

Comment:
Can't we allow the new `remote` and `local` options to be used with any link mode?

Reply:
+1

Reply:
It's possible. @lukeknep's feedback was to not have multiple ways of doing the same thing. Personally, I'd prefer to consolidate on only having `remote` and `local` but that entails a deprecation process and questions about how consistent the CLI and terraform are wrt to data modeling.

If we're going to keep `destination` and `source` here and in terraform I'm not sure how valuable it is to allow other link modes to also use `remote` and `local`.

Reply:
+1 for consolidating on `remote` and `local` and deprecating `destination` and `source`! Can we do this in Terraform too @linouk23?

Reply:
Per our last discussion we decided not to because it's quite a lot of effort required of our customers. It's also more work on the terraform implementation side. 

A bit of an unfortunate situation :(

Reply:
Also to be fair this isn't a one-way door decision. We should be able to move to just using `remote` and `local` in the future if we decide on that. I think we'll want to go with this for now to meet our deadlines.

Reply:
> We should be able to move to just using remote and local in the future if we decide on that.

Keep me updated! If we want to do this, it would be great to do it before our v4 release at the end of the year.

Reply:
> Per our last discussion we decided not to because it's quite a lot of effort required of our customers. It's also more work on the terraform implementation side.

+1